script/nicctl.sh

```diff
@@ -112,6 +113,18 @@ create_kvf() {
 	done
 }
 
+list_vf() {
+	vf_num=$(cat /sys/bus/pci/devices/"$1"/sriov_numvfs)
+	if [ "$vf_num" -eq 0 ]; then
+		echo "No VFs found for $1"
+		return
+	fi
+	for ((i = 0; i < vf_num; i++)); do
+		vfport=$(basename "$(readlink /sys/bus/pci/devices/"$1"/virtfn$i)")
```

Comment:
same here i think it's good to check if /sys/bus/pci/devices/"$1"/virtfn$i)"
Its a simple sanity check but makes the message much more understandable IMO 
Like Hay -> this doesn't exists